content/docs/reference-test-renderer.md

```diff
@@ -38,9 +38,9 @@ console.log(testRenderer.toJSON());
 //   children: [ 'Facebook' ] }
 ```
 
-You can use Jest's snapshot testing feature to automatically save a copy of the JSON tree to a file and check in your tests that it hasn't changed: [Learn more about it](https://facebook.github.io/jest/blog/2016/07/27/jest-14.html).
+Przy pomocy funkcjonalności biblioteki Jest do generowania snapshotów można automatycznie zapisać do pliku kopię drzewa w formacie JSON, a w teście sprawdzać, czy się ono nie zmieniło [(Więcej informacji na ten temat)](https://facebook.github.io/jest/blog/2016/07/27/jest-14.html).
```

Comment:
Czytałem to z 30 razy, i ciagle wydaje mi się niejasne, takie masło maślane. Sam nie mam pomysłu jak to zmienić ale może @iamarek @marcinostrouch @TeoTN @bartlomiejzuber @g12i coś zasugeruja?

Reply:
Nie wiem, co tutaj jest źle... 🤔